src/agent/src/mount.rs

```diff
@@ -288,6 +293,66 @@ async fn ephemeral_storage_handler(
     Ok("".to_string())
 }
 
+#[instrument]
+async fn nydus_storage_handler(
```

Comment:
Could you just introduce a `overlayfs` storage handler, and handle nydus rootfs type transparently to the guest? It can be done by constructing all overlayfs mount options at the runtime side, and it will be a generic overlayfs storage for the agent.

Reply:
Yes, it is a good idea, I will try